generators/ci-cd/templates/_.gitlab-ci.yml

```diff
@@ -24,39 +22,59 @@ stages:
     - release
     - deploy
 
-<%_ if (buildTool == 'gradle') { _%>
+<%_ if (buildTool === 'gradle') { _%>
 before_script:
     - export GRADLE_USER_HOME=`pwd`/.gradle
+    - chmod +x gradlew
```

Comment:
This line is also in gitlab-ci official Gradle template: https://gitlab.com/gitlab-org/gitlab-ci-yml/blob/master/Gradle.gitlab-ci.yml